sites/svelte.dev/content/blog/2023-02-01-whats-new-in-svelte-february-2023.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "What's new in Svelte: February 2023"
+description: "Minor versions and major satisfaction"
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+Since last month, we've seen a couple of minor versions across Svelte and SvelteKit, a bunch of educational content to support the 1.0 release, as well as the [State of JS survey](https://stateofjs.com) results...
+
+Svelte, once again, increased in satisfaction and interest, so thanks to everyone who voted!
+
+Let's dive into the updates...
+
+## What's new in SvelteKit
```

Comment:
we could add a bit of text to explain why there's so many fewer features this month than last.
```suggestion
## What's new in SvelteKit
Work in the Svelte ecosystem this month has been focused on SvelteKit bug fixes and addressing feedback from the 1.0 launch. One of the more awaited [bug fixes to SvelteKit's base path support](https://github.com/sveltejs/kit/issues/2958) landed, which means that you can now create SvelteKit projects on remote development environments. There have been several [new releases](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md) with these fixes as well as a handful of new features such as:
```